doc/devel/MEP/MEP12.rst

```diff
@@ -62,6 +62,7 @@ Gallery sections
 The naming of sections is critical and will guide the clean-up
 effort. The current sections are:
 
+* Live example (experimental)
```

Comment:
This document is pretty historical; I don't know that it needs updating.

Reply:
I'd be in favor of removing it 👍🏽

Reply:
Makes sense, removed in 55fe769c028a53a5defeb78a45d4e6f33c634187